app/controllers/v0/my_va/submission_statuses_controller.rb

```diff
@@ -53,12 +73,30 @@ def forms_based_on_feature_toggle
         restricted_list_of_forms
       end
 
+      def gateway_options_for_user
+        {
+          # ALWAYS enable benefits intake for backward compatibility
+          # The feature flag only controls whether to show ALL forms vs restricted list
+          benefits_intake_enabled: true,
+          decision_reviews_enabled: display_decision_reviews_forms?
+        }
+      end
+
       def display_all_forms?
+        # When this flag is true, show ALL forms without restriction (pass nil for allowed_forms)
+        # When false, show the restricted list of forms
```

Comment:
I think we could also consider changing the name of this feature flag since it seems that all benefits intake forms are enabled by default; with the refactor, when this flag is true it would show ALL forms without restriction.

Reply:
Ah ok, I understand better now - thanks for the clarification